projects/GEDKeeper2/GKUI/BaseWin.cs

```diff
@@ -1376,55 +1455,119 @@ public bool RecordDelete(GEDCOMRecord record, bool confirm)
             {
                 //string xref = record.XRef;
                 string msg = "";
-
                 switch (record.RecordType)
                 {
                     case GEDCOMRecordType.rtIndividual:
-                        msg = string.Format(LangMan.LS(LSID.LSID_PersonDeleteQuery), ((GEDCOMIndividualRecord)record).GetNameString(true, false));
+                    {
+                        msg = string.Format(LangMan.LS(
+                            LSID.LSID_PersonDeleteQuery),
+                            ((GEDCOMIndividualRecord)record).GetNameString(
+                                true, false));

```

Comment:
I'm sorry. I just don't understand, in terms of readability, when at least the first and relatively short argument divorced from the call.
It is possible and on another to write:
```
msg = string.Format(
    LangMan.LS(LSID.LSID_PersonDeleteQuery),
    ((GEDCOMIndividualRecord)record).GetNameString(true, false));
```
What here bad?